### DISCUSSION TITLE
Just some positive feedback

### DISCUSSION THREAD

**Discussion by grab-a-byte** [vlang/v#16517] [General]
Hi guys, hope the core team see this. I just wanted to say from what I have seen of V. It looks like it will fix a lot of what i hate about golang (as i know syntax is similar) and I cant wait until this hits 1.0 with great tooling, one click installers and it can become a very simple part of my OS setup scripts.
Please keep up the good work and i look forward to seeing this language come to fruition 😊

**Comment by medvednikov** [comment]
Thanks :) Glad you like it.

**Comment by Delta456** [comment]
Thank you for your support 😄

**Comment by ghost** [comment]
Hi everyone..
Iam new to vlang😀


How to get v lang book pdf.if any one have a pdf plz share with me 😀..

**Reply by JalonSolov** [reply]
There is no pdf.

You can purchase the paper version or the eBook version by clicking the link on the home page.

If someone has created a pdf from the eBook, that is illegal to share.

**Reply by ghost** [reply]
It's hard to read "V-lang" books.Is there any tutorial videos.

**Comment by grab-a-byte** [comment]
The only query I have regarding this is if we have a timeline for intellisense/autocompletions in various IDEs? (mainly vs code for myself) It's the only thing making me not deep dive in v but if it's there it's very much a candidate for my advent of code next year (I use advent of code to learn new languages). Haven't had a langauge get my excited for a while now xD

**Reply by JalonSolov** [reply]
Intellisense works with vscode and the v plugin there, but there are issues on Windows.  The Intellij plugin (with intellisense) will be going to open Beta on 1/1/2023.  I don't know of any others.

For advent of code, check out https://github.com/vlang/adventofcode

**Reply by grab-a-byte** [reply]
Hopefully windows issues fixed soon! (I use windows becasue i game also) But good to hear about intelliJ too, could be nice. I've been keeping check on that repo but thanks for the link :)
